avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/TensorMetadataGeneration.scala

```diff
@@ -93,7 +97,13 @@ class TensorMetadataGeneration {
    */
   private def getColsWithHashInfoCardinalityMapping(params: TensorizeInParams): Map[String, Long] = {
 
-    TensorizeInConfigHelper.getColsHashInfo(params).map(x => x._1 -> x._2.hashBucketSize.toLong)
+    TensorizeInConfigHelper.concatFeaturesAndLabels(params)
```

Comment:
why not reuse `TensorizeInConfigHelper.getColsHashInfo`?

Reply:
That's definitely better. Not sure what I was thinking.